### DISCUSSION TITLE
Angular Version Upgrade

### DISCUSSION THREAD

**Discussion by mudassarlodhi** [DavideViolante/Angular-Full-Stack#276] [General]
Angular v17 is out. not sure whether the project can be directly upgraded to v17 or not. APMK this may not be recommended.
This can be looked into and discussed. Based on the dependencies this project uses, it can be upgraded to next possible angular version

**Comment by DavideViolante** [comment]
Updated

**Reply by mudassarlodhi** [reply]
Awesome. that was quick
